app/validators/casa_org_validator.rb

```diff
@@ -0,0 +1,38 @@
+class CasaOrgValidator < ActiveModel::Validator
+  VALID_TWILIO_PHONE_NUMBER_LENGTH = 12
+  VALID_TWILIO_COUNTRY_CODE = "+1"
+
+  def validate(record)
+    valid_twilio_phone_number(record.twilio_phone_number, record)
+  end
+
+  private
+
+  def valid_twilio_phone_number(number, record)
```

Comment:
@harsohailB Can this method be DRY'd up? It seems awfully similar to the phone number validation method for validators/user_validator.rb

Reply:
Will do!

Reply:
Abstracted out phone number validity logic into a helper named `PhoneNumberHelper`. Will create a ticket to point our previously worked on `UserValidator` and importers to this as well for code reuse.